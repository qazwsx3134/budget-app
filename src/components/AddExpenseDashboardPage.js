import React from "react";


const AddExpenseDashboardPage = ()=>(
    <div>
        This is from my add dashboard component
    </div>
);

export default  AddExpenseDashboardPage;