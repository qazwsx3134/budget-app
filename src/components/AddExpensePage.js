import React from 'react';
import ExpenseForm from "./ExpenseForm";
import { connect } from "react-redux";
import { addExpense } from "../actions/expenses";


const AddExpensePage = (props) => ( //必須把props丟進來 因為connect把dispatch 整合到props裡
  <div>
    <h1>Add Expense</h1>
    <ExpenseForm 
      onSubmit={(expense)=>{ //把這個fuction 作為props傳下去
        props.dispatch(addExpense(expense))//expense是ExpenseForm 傳上來的object
        props.history.push('/')//導向到path dashboard
      }}
    />
  </div>
);

export default connect()(AddExpensePage)
