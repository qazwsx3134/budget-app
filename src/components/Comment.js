import React ,{useState} from 'react';
import { connect } from "react-redux";
import { makeStyles } from '@material-ui/core/styles';
import Grid from '@material-ui/core/Grid';
import List from '@material-ui/core/List';
import ListItem from '@material-ui/core/ListItem';
import Divider from '@material-ui/core/Divider';
import ListItemText from '@material-ui/core/ListItemText';
import ListItemAvatar from '@material-ui/core/ListItemAvatar';
import Avatar from '@material-ui/core/Avatar';
import Typography from '@material-ui/core/Typography';
import Box from '@material-ui/core/Box';
import Modal from '@material-ui/core/Modal';
import Backdrop from '@material-ui/core/Backdrop';
import Fade from '@material-ui/core/Fade';
import PersonIcon from '@material-ui/icons/Person';
import dayjs from "dayjs";

const useStyles = makeStyles({
    root: {
      width: '100%',
      maxWidth: 1015,
    },
    inline: {
      display: 'inline',
    },  
    avatar: {
        color: '#FF6868',
        backgroundColor:"#FFCECE",
      },
    
    modal: {
      display: 'flex',
      alignItems: 'center',
      justifyContent: 'center',
    },
    grid:{
      marginLeft: 30
    }
  });

  export const Comment = (props)=> {

    const [open, setOpen] = React.useState(false);
     
    const classes = useStyles();
    
    const handleOpen = () => {
      setOpen(true);
    };

    const handleClose = () => {
      setOpen(false);
    };
    return (
      <div className={classes.grid}>
      
      <Grid container key={props._id} >
        <Grid  item xs={12} >
        <ListItem  alignItems="flex-start">
            <ListItemAvatar>
              <Avatar className={classes.avatar}>
                <PersonIcon/>
              </Avatar>
            </ListItemAvatar>
            <ListItemText
              
              primary={
                <React.Fragment>
                  <Typography
                    component="span"
                    variant="h6"
                    className={classes.inline}
                    color="primary"
                  >
                    {props.name}
                  </Typography>

                  <Typography
                    component="span"
                    variant="h6"
                    color="textPrimary"
                  >
                    
                  </Typography>

                  </React.Fragment>
                  }
              secondary={
                <React.Fragment>
                  <Typography
                    component="div"
                    className={classes.inline}
                    color="textPrimary"
                  >
                  {props.text.split('\n').map((i,key)=>{
                    return <Box key={key}  component="p" fontSize="h6.fontSize" fontWeight="fontWeightRegular" m={1}>&nbsp;&nbsp;{i}</Box>;
                  })}
                    {/* <Box component="div" fontSize="h6.fontSize" lineHeight={1} fontWeight="fontWeightRegular"  m={1}>
                    &nbsp;&nbsp;{props.text}
                    </Box> */}
                    <Box component="div" fontSize={14}  fontWeight="fontWeightLight"  >
                    Ôºç{dayjs(props.posted).format('MMMM D YYYY, h:mm a')}
                    </Box>
                    
                  </Typography>
                  
                </React.Fragment>
              }
            />
          </ListItem>
        </Grid>
        <Grid item xs={12} >
        <div className="list-item__border__qanda">
        {(props.otherPic.map((otherPic)=>{
                        return (
                            <div key={otherPic._id}>
                            <input 
                                className="resize-image"
                                type="image"
                                src={`${otherPic.pictureURL}`}
                                alt="Image" 
                                name="Image_popup" 
                                onClick={handleOpen} />
                            <Modal
                                aria-labelledby="transition-modal-title"
                                aria-describedby="transition-modal-description"
                                className={classes.modal}
                                open={open}
                                onClose={handleClose}
                                closeAfterTransition
                                BackdropComponent={Backdrop}
                                BackdropProps={{
                                    timeout: 300,
                                }}
                            >
                            <Fade in={open}>
                                <div >
                                <img className="responsive-image" src={`${otherPic.pictureURL}`}/>
                                </div>
                            </Fade>
                            </Modal>
                            </div>
                        )})) 
                         }
                </div>  
        </Grid>
      </Grid>

      
      <Divider  component="li" variant="inset"/>
          </div>
    );
  };

