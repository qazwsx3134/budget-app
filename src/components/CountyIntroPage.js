import React from 'react';
import { connect } from "react-redux";
import HeaderPage from "./HeaderPage";



export class CountyIntroPage extends React.Component{
 
 render()
  {
    return (
    <div>
      <HeaderPage />
      <div className="content-container">
      </div>
      
    </div>
  )}
}




const mapStateToProps = (state) =>{
  return {
    viewPoints: state.viewpoint
  }
};

export default connect(mapStateToProps)(CountyIntroPage)
