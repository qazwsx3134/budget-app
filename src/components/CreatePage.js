import React from 'react';
import AppBar from '@material-ui/core/AppBar';
import Button from '@material-ui/core/Button';
import Card from '@material-ui/core/Card';
import CardActions from '@material-ui/core/CardActions';
import CardContent from '@material-ui/core/CardContent';
import CardHeader from '@material-ui/core/CardHeader';
import CssBaseline from '@material-ui/core/CssBaseline';
import Grid from '@material-ui/core/Grid';
import StarIcon from '@material-ui/icons/StarBorder';
import Toolbar from '@material-ui/core/Toolbar';
import Typography from '@material-ui/core/Typography';
import Link from '@material-ui/core/Link';
import { makeStyles } from '@material-ui/core/styles';
import Container from '@material-ui/core/Container';
import Box from '@material-ui/core/Box';

function Copyright() {
  return (
    <Typography variant="body2" color="textSecondary" align="center">
      {'Copyright Â© '}
      <Link color="inherit" href="https://material-ui.com/">
        Your Website
      </Link>{' '}
      {new Date().getFullYear()}
      {'.'}
    </Typography>
  );
}

const useStyles = makeStyles(theme => ({
  '@global': {
    ul: {
      margin: 0,
      padding: 0,
    },
    li: {
      listStyle: 'none',
    },
  },
  appBar: {
    borderBottom: `1px solid ${theme.palette.divider}`,
  },
  toolbar: {
    flexWrap: 'wrap',
  },
  toolbarTitle: {
    flexGrow: 1,
  },
  link: {
    margin: theme.spacing(1, 1.5),
  },
  heroContent: {
    padding: theme.spacing(8, 0, 6),
  },
  cardHeader: {
    backgroundColor:
      theme.palette.type === 'dark' ? theme.palette.grey[700] : theme.palette.grey[200],
  },
  cardPricing: {
    display: 'flex',
    justifyContent: 'center',
    alignItems: 'baseline',
    marginBottom: theme.spacing(2),
  },
  footer: {
    borderTop: `1px solid ${theme.palette.divider}`,
    marginTop: theme.spacing(8),
    paddingTop: theme.spacing(3),
    paddingBottom: theme.spacing(3),
    [theme.breakpoints.up('sm')]: {
      paddingTop: theme.spacing(6),
      paddingBottom: theme.spacing(6),
    },
  },
}));

const tiers = [
  {
    title: 'Event',
    note: 'N/A',
    description: ['1', '2', '3', '4'],
    buttonText: 'Post Event',
    buttonVariant: 'contained',
    url: '/postevent',
  },
  {
    title: 'Place',
    subheader: 'Most popular',
    note: 'N/A',
    description: [
      '20 users included',
      '10 GB of storage',
      'Help center access',
      'Priority email support',
    ],
    buttonText: 'Post Place',
    buttonVariant: 'contained',
    url: '/postplace',
  },
  {
    title: 'Food',
    note: 'N/A',
    description: [
      '50 users included',
      '30 GB of storage',
      'Help center access',
      'Phone & email support',
    ],
    buttonText: 'Post Food',
    buttonVariant: 'contained',
    url: '/postfood',
  },
];

export default function CreatePage() {
  const classes = useStyles();

  return (
    <React.Fragment>
      <CssBaseline />
      <AppBar position="static" color="default" elevation={0} className={classes.appBar}>
        <Toolbar className={classes.toolbar}>
          <Typography variant="h6" color="inherit" noWrap className={classes.toolbarTitle}>
            CreatePage
          </Typography>
          <nav>
            <Link variant="button" color="textPrimary" href="#" className={classes.link}>
              Not in use
            </Link>
            <Link variant="button" color="textPrimary" href="#" className={classes.link}>
            Not in use
            </Link>
            <Link variant="button" color="textPrimary" href="#" className={classes.link}>
            Not in use
            </Link>
          </nav>
        </Toolbar>
      </AppBar>
      {/* Hero unit */}
      <Container maxWidth="sm" component="main" className={classes.heroContent}>
        <Typography component="h1" variant="h2" align="center" color="textPrimary" gutterBottom>
          Post the Nice stuff
        </Typography>
        <Typography variant="h5" align="center" color="textSecondary" component="p">
          Event Food Places
        </Typography>
      </Container>
      {/* End hero unit */}
      <Container maxWidth="md" component="main">
        <Grid container spacing={5} alignItems="flex-end">
          {tiers.map(tier => (
            // Enterprise card is full width at sm breakpoint
            <Grid item key={tier.title} xs={12} sm={tier.title === 'Enterprise' ? 12 : 6} md={4}>
              <Card>
                <CardHeader
                  title={tier.title}
                  subheader={tier.subheader}
                  titleTypographyProps={{ align: 'center' }}
                  subheaderTypographyProps={{ align: 'center' }}
                  action={tier.title === 'Pro' ? <StarIcon /> : null}
                  className={classes.cardHeader}
                />
                <CardContent>
                  <div className={classes.cardPricing}>
                    <Typography component="h2" variant="h3" color="textPrimary">
                      {tier.note}
                    </Typography>
                    <Typography variant="h6" color="textSecondary">
                      
                    </Typography>
                  </div>
                  <ul>
                    {tier.description.map(line => (
                      <Typography component="li" variant="subtitle1" align="center" key={line}>
                        {line}
                      </Typography>
                    ))}
                  </ul>
                </CardContent>
                <CardActions>
                  
                  <Button fullWidth href={`${tier.url}`} variant={tier.buttonVariant} color="primary">
                    {tier.buttonText}
                  </Button>
                </CardActions>
              </Card>
            </Grid>
          ))}
        </Grid>
      </Container>
    </React.Fragment>
  );
}