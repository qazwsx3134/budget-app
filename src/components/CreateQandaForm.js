
import React,{ useState } from 'react';
import { useDispatch,connect } from 'react-redux'
import Avatar from '@material-ui/core/Avatar';
import Button from '@material-ui/core/Button';
import CssBaseline from '@material-ui/core/CssBaseline';
import TextField from '@material-ui/core/TextField';
import FormControlLabel from '@material-ui/core/FormControlLabel';
import Checkbox from '@material-ui/core/Checkbox';
import Link from '@material-ui/core/Link';
import Grid from '@material-ui/core/Grid';
import Box from '@material-ui/core/Box';
import CloudUploadIcon from '@material-ui/icons/CloudUpload';
import Typography from '@material-ui/core/Typography';
import { makeStyles } from '@material-ui/core/styles';
import Container from '@material-ui/core/Container';
import { apiUserLogin } from "../api/api";
import { login } from "../actions/auth";
import { useHistory } from "react-router-dom";
import MenuItem from '@material-ui/core/MenuItem';
import HelpIcon from '@material-ui/icons/Help';
import { apiUploadSingle,apiUploadMulti,apiPostQanda,apiDeleteSingle } from "../api/api";
import { addQanda } from "../actions/qanda";



  

function Copyright() {
  return (
    <Typography variant="body2" color="textSecondary" align="center">
      {'Copyright Â© '}
      <Link color="inherit" href="https://material-ui.com/">
        Wayfarer
      </Link>{' '}
      {new Date().getFullYear()}
      {'.'}
    </Typography>
  );
}

const useStyles = makeStyles(theme => ({
  paper: {
    marginTop: theme.spacing(8),
    display: 'flex',
    flexDirection: 'column',
    alignItems: 'center',
  },
  avatar: {
    margin: theme.spacing(1),
    backgroundColor: theme.palette.secondary.main,
  },
  form: {
    width: '100%', // Fix IE 11 issue.
    marginTop: theme.spacing(1),
  },
  submit: {
    margin: theme.spacing(3, 0, 2),
  },
  errorMessage:{
    color: '#FF0000'
  },
  menu: {
    width: 200,
  },
  textField: {
    marginLeft: theme.spacing(1),
    marginRight: theme.spacing(1),
    width: 200,
  },
  button: {
    margin: theme.spacing(1),
  },
  input: {
    display: 'none',
  },
}));

  function CreateQandaForm(props) {

    const topics = [
      {
        value: 'For-Sale',
      },
      {
        value: 'Question',
      },
      {
        value: 'Team-Up',
      },
      {
        value: 'Discussion',
      },
      {
        value: 'Chat',
      },
    ]

  const classes = useStyles();
  let history = useHistory();
  const dispatch = useDispatch()

  const [errorMessage, setErrorMessage]= useState();
  const [name, setName] = useState('');
  const [title, setTitle] = useState('');
  const [topic, setTopic]= useState('');
  const [city, setCity] = useState('');
  const [text, setText] = useState('');
  const [pictureURL, setPictureURL ] = useState('');
  const [otherPic ,setOtherPic] = useState([]);
  
  
  
  const onDeleteOtherPicture = async(e)=>{

    
    const data = {"key":e.currentTarget.id}
    
    const result = otherPic.filter((url)=> url.pictureURL !== e.currentTarget.value)
    const deleteRes = await apiDeleteSingle(data)

    setOtherPic(result)
  }

  const onTopicChange = (e)=>{
    setTopic(e.target.value)
  }
  const onCityChange = (e)=>{
    setCity(e.target.value)
  }
  const onNameChange = (e)=>{
    setName(e.target.value)
  }
  const onTitleChange = (e)=>{
    setTitle(e.target.value)
  }
  const onAddressChange = (e)=>{
    setAddress(e.target.value)
  }
  const onInstaURLChange = (e)=>{
    setInstaURL(e.target.value)
  }
  const onWebsiteChange = (e)=>{
    setWebsite(e.target.value)
  }
  const onMapChange = (e)=>{
    setMap(e.target.value)
  }
  const onTextChange = (e)=>{
    setText(e.target.value)
  }
  const onHourstayChange = (e)=>{
    setHourstay(e.target.value)
  }
  const onBusinessHoursChange = (e)=>{
    setBusinessHours(e.target.value)
  }
  const onAveragePriceChange = (e)=>{
    setAveragePrice(e.target.value)
  }
  
  const onUploadSingleChange = async(e)=>{
    // single pic uplaod
    const formData = new FormData();
    formData.append('image',e.target.files[0])
    const picRes = await apiUploadSingle(formData)
    setPictureURL(picRes.data)

  }
  const onUploadMultiChange = async(e)=>{
    const multiFormData = new FormData()
    for (let index = 0; index < e.target.files.length; index++) {
      multiFormData.append('image',e.target.files[index])
      
    }
    const multiRes = await apiUploadMulti(multiFormData)
    setOtherPic(otherPic.concat(multiRes.data) )
  }

  const onSubmit = async(e) =>{
    


    const fullName = `${props.auth.firstName} ${props.auth.lastName}`
    
    const eventdata= {
      "name": props.auth.firstName,
      "title":title,
      "city":props.city,
      "type": "qanda",
      "topic":topic,
      "text":text,
      "author":[{user:props.auth._id,"name": fullName}],
      "otherPic":otherPic,
      
    }
    if (!topic) {
      setErrorMessage('You have to choose a topic')
      event.preventDefault();
    }
    if (props.auth.firstName) {
        
      const postRes = await apiPostQanda(eventdata)

        console.log(postRes);
      dispatch(addQanda(eventdata))
      
      return
    }
    setErrorMessage('You have to login to comment')
    
  }
  return (
    <Container component="main" maxWidth="sm">
      <CssBaseline />
      <div className={classes.paper}>
        <Avatar className={classes.avatar}>
          <HelpIcon />
        </Avatar>
        <Typography component="h1" variant="h5">
          Ask a Question
        </Typography>
        <Typography component="h5" variant="subtitle2" className={classes.errorMessage}>
          {errorMessage 
            ? errorMessage
            : null}
        </Typography>
        <form className={classes.form} onSubmit={onSubmit}>

        {/* Label selector */}
        <TextField
          id="standard-select-city"
          select
          variant="outlined"
          label="Topic"
          required
          className={classes.textField}
          onChange={onTopicChange}
          value={topic}
          SelectProps={{
            MenuProps: {
              className: classes.menu,
            },
          }}
          helperText="Please select Topic"
          margin="normal"
        >
          {topics.map(option => (
            <MenuItem key={option.value} value={option.value}>
              {option.value}
            </MenuItem>
          ))}
        </TextField>
        
    
        {/* Name */}
          <TextField
            variant="outlined"
            margin="normal"
            required
            fullWidth
            id="name"
            label="Name"
            name="name"
            autoComplete="name"
            value={props.auth.firstName ? props.auth.firstName : 'You must login for submit'}
            disabled
            onChange={onNameChange}
          />
          {/* Title */}
          <TextField
            variant="outlined"
            margin="normal"
            fullWidth
            name="Title"
            label="Title"
            type="Title"
            id="Title"
            autoComplete="current-organizer"
            onChange={onTitleChange}
          />
        {/* text */}
        <TextField
          id="text"
          name="text"
          label="Text"
          variant="outlined"
          margin="normal"
          fullWidth
          value={text}
          onChange={onTextChange}
          multiline
          rows="4"
        />
        
        {/* otherPic */}
        <input
            name="image"
            accept="image/*"
            className={classes.input}
            id="contained-button-multifile"
            type="file"
            multiple
            onChange={onUploadMultiChange}
        />
        <label htmlFor="contained-button-multifile">
        <Button
            variant="contained"
            color="default"
            className={classes.button}
            startIcon={<CloudUploadIcon />}
            component="span"
        >
            Upload Pictures
        </Button>
        </label>
        
        { otherPic && otherPic.map((otherPic)=>{
                        return(
                        <div key={`${otherPic.key}`}>
                          <img className="responsive-image" src={`${otherPic.pictureURL}`}/>
                          <Button
                          variant="contained"
                          color="primary"
                          id={`${otherPic.key}`}
                          value={`${otherPic.pictureURL}`}
                          className={classes.submit}
                          onClick={onDeleteOtherPicture}
                        >
                        Delete Picture
                        </Button>
                        </div>)
        })}
        <Typography component="h5" variant="subtitle2" className={classes.errorMessage}>
          {errorMessage 
            ? errorMessage
            : null}
        </Typography>
          <Button
            type="submit"
            fullWidth
            variant="contained"
            color="primary"
            className={classes.submit}
          >
            Submit
          </Button>
        </form>
      </div>
      <Box mt={8}>
        <Copyright />
      </Box>
    </Container>
  );
}

const mapStateToProps = (state)=>{
  return {
      auth: state.auth
  };
}
export default connect(mapStateToProps)(CreateQandaForm);