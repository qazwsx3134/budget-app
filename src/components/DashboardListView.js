import React,{useEffect} from 'react';
import { makeStyles } from '@material-ui/core/styles';
import { useDispatch  } from "react-redux";
import { sortByLike} from "../actions/filters";
import Card from '@material-ui/core/Card';
import CardActionArea from '@material-ui/core/CardActionArea';
import CardActions from '@material-ui/core/CardActions';
import CardContent from '@material-ui/core/CardContent';
import CardMedia from '@material-ui/core/CardMedia';
import Button from '@material-ui/core/Button';
import Typography from '@material-ui/core/Typography';
import IconButton from '@material-ui/core/IconButton';
import FavoriteIcon from '@material-ui/icons/Favorite';
import CardHeader from '@material-ui/core/CardHeader';
import ShareIcon from '@material-ui/icons/Share';
import Grid from '@material-ui/core/Grid';
import {
  useParams,
  Link,
} from "react-router-dom";
import Box from '@material-ui/core/Box';
import FlashOnOutlinedIcon from '@material-ui/icons/FlashOnOutlined';
import ThumbUpIcon from '@material-ui/icons/ThumbUp';

const useStyles = makeStyles({
  card: {
    borderRadius: 8,
    marginTop: 10,
    maxWidth: 320,
  },
  cardDetails: {
    margin: 5,
  },
  media: {
    borderRadius: 8,
    marginBottom: 6,
    position: 'static',
    height: '100%',
    width: '100%',
    objectfit: 'cover',
    paddingTop: '133.33%', // 16:9
  },
  nopadding:{
    padding:0,
  },
  textalign:{
    margin: 2,
    paddingLeft:5,
  }
});

export default function DashboardListView({history,pictureURL,name, _id, city, englishname, description,label,address,website,type,likeby,dislikeby}) {
  
  const classes = useStyles();
  
  
  return (
    <Grid item key={_id} xs={6} md={3}>
    <Card className={classes.card} elevation={0}>
      <Link className="button-no-deco" to={`${history.location.pathname}/${type}/${_id}`}>
      <CardActionArea >
        
        <CardMedia
        className={classes.media}
        image={`${pictureURL.pictureURL}`}
        title={englishname}
      />
      <Typography className={classes.textalign} display="block" color="textPrimary" component="div" gutterBottom>
            <Box fontSize={14}  component="span" fontWeight="fontWeightBold" >
              {city.toUpperCase()}&nbsp;&nbsp;
            </Box>
            <Box fontSize={14} alignItems="center" component="span" fontWeight="fontWeightBold"  >    
              &nbsp;|&nbsp;
              {likeby.length}&nbsp;LIKES
              &nbsp;
              {dislikeby.length}&nbsp;BOO!&nbsp;|
            </Box>
            
      </Typography>

      <Typography className={classes.textalign}  display="block" color="textPrimary" component="div"  >
          <Box fontFamily="fontFamily"  fontSize={18} fontWeight="fontWeightBold" >
          {englishname}
          </Box>
      </Typography>
 
        
      </CardActionArea>
      </Link>

    </Card>
    </Grid>
  );
}