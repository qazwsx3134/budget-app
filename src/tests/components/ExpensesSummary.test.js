import React from "react";
import { shallow } from "enzyme";
import { ExpensesSummary } from "../../components/ExpensesSummary";

test('should right render 1 expense', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={1} expensesTotal={200} />)
    expect(wrapper).toMatchSnapshot();
});


test('should right render 2 expenses', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={2} expensesTotal={3300} />)
    expect(wrapper).toMatchSnapshot();
});