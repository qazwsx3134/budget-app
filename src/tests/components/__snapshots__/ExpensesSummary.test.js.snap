// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should right render 1 expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totaling 
    $2.00
     
  </h1>
</div>
`;

exports[`should right render 2 expenses 1`] = `
<div>
  <h1>
    Viewing 
    2
     
    expenses
     totaling 
    $33.00
     
  </h1>
</div>
`;
